dsbridge/src/main/java/wendu/dsbridge/DWebView.java

```diff
@@ -120,6 +121,7 @@ void init() {
         settings.setLoadWithOverviewMode(true);
         settings.setSupportMultipleWindows(true);
         settings.setAppCachePath(APP_CACAHE_DIRNAME);
+        settings.setMediaPlaybackRequiresUserGesture(false);
```

Comment:
why this have been added?

Reply:
because we need to enable js api to play automatically. should we add it in application code?